### DISCUSSION TITLE
Issue with React Navigation & KeyboardAvoidingView

### DISCUSSION THREAD

**Discussion by jonasmerlin** [react-navigation/react-navigation#10509] [Q&A]
Hey! I'm using React Navigation to show two stack screens, each of which is wrapped in a `KeyboardAvoidingView` with behaviour "position". Since the navigation header is shown, I set `keyboardVerticalOffset` on the avoiding view to be the header's height (gotten from the `useHeaderHeight()` hook), so that the whole screen is moved above the keyboard. This works fine. But when I put in text on the first screen and click the button to navigate to the second without closing the keyboard first, the second screen is moved upward by the height of the header for some reason.

When I take the `keyboardVerticalOffset` out, this doesn't happen. But then, the avoiding behavior is not what I want.

I already tried setting `enabled={isFocused}` where is focused is the return value of `useIsFocused()` from React Navigation. But this doesn't work as well. (It only works when I set it to false on the _focused_ screen but ... then that's not what I want, of course :D)

Any ideas?

**Comment by raajnadar** [comment]
Did you try other options like "padding" and "height" instead of "position"?
